cmd/admin-user-sts-info.go

```diff
@@ -0,0 +1,131 @@
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/fatih/color"
+	"github.com/minio/cli"
+	json "github.com/minio/colorjson"
+	"github.com/minio/mc/pkg/probe"
+	"github.com/minio/pkg/console"
+	iampolicy "github.com/minio/pkg/iam/policy"
+)
+
+var adminUserSTSAcctSubcommands = []cli.Command{
+	adminUserSTSAcctInfoCmd,
+}
+
+var adminUserSTSAcctCmd = cli.Command{
+	Name:            "sts",
+	Usage:           "manage STS accounts",
+	Action:          mainAdminUserSTSAcct,
+	Before:          setGlobalsFromContext,
+	Flags:           globalFlags,
+	Subcommands:     adminUserSTSAcctSubcommands,
+	HideHelpCommand: true,
+}
+
+// mainAdminUserSTSAcct is the handle for "mc admin user sts" command.
+func mainAdminUserSTSAcct(ctx *cli.Context) error {
+	commandNotFound(ctx, adminUserSTSAcctSubcommands)
+	return nil
+}
+
+var adminUserSTSAcctInfoFlags = []cli.Flag{
+	cli.BoolFlag{
+		Name:  "policy",
+		Usage: "print policy in JSON format",
+	},
+}
+
+var adminUserSTSAcctInfoCmd = cli.Command{
+	Name:         "info",
+	Usage:        "display temporary account info",
+	Action:       mainAdminUserSTSAcctInfo,
+	OnUsageError: onUsageError,
+	Before:       setGlobalsFromContext,
+	Flags:        append(adminUserSTSAcctInfoFlags, globalFlags...),
+	CustomHelpTemplate: `NAME:
+  {{.HelpName}} - {{.Usage}}
+
+USAGE:
+  {{.HelpName}} ALIAS STS-ACCOUNT
+
+FLAGS:
+  {{range .VisibleFlags}}{{.}}
+  {{end}}
+EXAMPLES:
+  1. Display information for the temporary account 'J123C4ZXEQN8RK6ND35I'
+     {{.Prompt}} {{.HelpName}} myminio/ J123C4ZXEQN8RK6ND35I
+`,
+}
+
+// checkAdminUserSTSAcctInfoSyntax - validate all the passed arguments
+func checkAdminUserSTSAcctInfoSyntax(ctx *cli.Context) {
+	if len(ctx.Args()) != 2 {
+		showCommandHelpAndExit(ctx, 1)
+	}
+}
+
+// mainAdminUserSTSAcctInfo is the handle for "mc admin user sts info" command.
+func mainAdminUserSTSAcctInfo(ctx *cli.Context) error {
+	checkAdminUserSTSAcctInfoSyntax(ctx)
+
+	console.SetColor("AccountMessage", color.New(color.FgGreen))
+
+	// Get the alias parameter from cli
+	args := ctx.Args()
+	aliasedURL := args.Get(0)
+	stsAccount := args.Get(1)
+
+	// Create a new MinIO Admin Client
+	client, err := newAdminClient(aliasedURL)
+	fatalIf(err, "Unable to initialize admin connection.")
+
+	stsInfo, e := client.TemporaryAccountInfo(globalContext, stsAccount)
+	fatalIf(probe.NewError(e).Trace(args...), "Unable to get information of the specified service account")
+
+	if ctx.Bool("policy") {
+		if stsInfo.Policy == "" {
+			fatalIf(errDummy().Trace(args...), "No policy found associated to the specified service account. Check the policy of its parent user.")
+		}
+		p, e := iampolicy.ParseConfig(strings.NewReader(stsInfo.Policy))
+		fatalIf(probe.NewError(e).Trace(args...), "Unable to parse policy.")
+		enc := json.NewEncoder(os.Stdout)
+		enc.SetIndent("", " ")
+		fatalIf(probe.NewError(enc.Encode(p)).Trace(args...), "Unable to write policy to stdout.")
+		return nil
+	}
+
+	fmt.Println(stsInfo.ImpliedPolicy)
```

Comment:
```suggestion
	fmt.Println(stsInfo.ImpliedPolicy)
```

is this json formatted already? this is an odd place to print something like this before you print the actual info

Reply:
yes, this was a typo